app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_cp, file_name, file_to_copy = command.split()
```

Comment:
Before validate that command.split() have exactly 3 parameters
Because you code will broke if command will be
1. cp file.txt
2. cp file.txt file2.txt file3.txt